web/source/osk/visualKeyboard.ts

```diff
@@ -1073,42 +1001,15 @@ namespace com.keyman.osk {
 
     clearPopup() {
       // Remove the displayed subkey array, if any, and cancel popup request
-      if(this.subkeyPopup) {
-        this.subkeyPopup.clear();
-        this.subkeyPopup = null;
+      if(this.subkeyGesture) {
+        this.subkeyGesture.clear();
+        this.subkeyGesture = null;
       }
 
-      if(this.subkeyDelayTimer) {
-          window.clearTimeout(this.subkeyDelayTimer);
-          this.subkeyDelayTimer = null;
+      if(this.pendingSubkey) {
+        this.pendingSubkey.cancel();
+        this.pendingSubkey = null;
       }
-      this.popupBaseKey = null;
-    }
-
-    //#region 'native'-mode subkey handling
-    /**
-     * Display touch-hold array of 'sub-keys' above the currently touched key
-     * @param       {Object}    e      primary key element
-     */
-    showSubKeys(e: KeyElement) {
```

Comment:
Entirely subsumed by the new, encapsulated classes (within the `browser` sub-namespace).